src/haiway/files/memory.py

```
@@ -0,0 +1,270 @@
+
+        self._timestamps[normalized] = time.time()
+
+    @asynccontextmanager
+    async def open_file(
+        self,
+        path: Path,
+        mode: str = "r",
+        /,
+        *,
+        encoding: str | None = None,
+        buffering: int = -1,
```

Pylint (3.3.7)

[warning] Unused argument 'buffering'

(W0613)